Uniswap/interface
# fix: hotfix un-nest translations on AddLiquidityV2 page

## Description


The information message on the AddLiquidityV2 page is not being translated correctly. Instead of the actual message, users are seeing a random string (likely an internal lingui hash). The root cause is the use of a `Trans` component inside of another `Trans` component. Un-nesting them resolves the issue and allows the intended message to be shown.



_Linear ticket:_ https://linear.app/uniswap/issue/WEB-2783/weird-text-string-on-the-add-v2-liquidity-page-here
_Slack thread:_ https://uniswapteam.slack.com/archives/C047U65H422/p1693507380586379


## Screen capture

### Before
|    Mobile    |   Desktop   |
| ------------ | ----------- |
| <img width="405" alt="Screenshot 2023-08-31 at 6 08 51 PM" src="https://github.com/Uniswap/interface/assets/18626088/a0c01820-d062-400b-aae9-78be9b0ebd2f"> | <img width="487" alt="Screenshot 2023-08-31 at 6 08 34 PM" src="https://github.com/Uniswap/interface/assets/18626088/7767bc32-8f1b-4cac-b8a9-43135cb7dadf"> |

### After
|    Mobile    |   Desktop   |
| ------------ | ----------- |
| <img width="425" alt="Screenshot 2023-08-31 at 6 08 12 PM" src="https://github.com/Uniswap/interface/assets/18626088/1b3a947a-6cbc-4025-b005-b3426a01dca4">  | <img width="475" alt="Screenshot 2023-08-31 at 6 07 32 PM" src="https://github.com/Uniswap/interface/assets/18626088/673be59f-2645-4f75-b1c0-e9ed4bea2983"> |

## Test plan


### Reproducing the error


1. Go to the V2 AddLiquidity page
2. Information message appearing at the top of the page is a random hash

### QA (ie manual testing)
Replace the "Tip" call to action and following information message with another valid, previously translated string, e.g. "Pool." Running a production build locally, you will see that, when the strings are nested, the message is incorrect. When they are un-nested, the message is correct.

### Automated testing
